libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1220,15 +1220,17 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     // @Field: PErr: difference between estimated potential energy and desired potential energy
     // @Field: EDelta: current error in speed/balance weighting
     // @Field: LF: aerodynamic load factor
-    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF",
-                       "s------",
-                       "F------",
-                       "Qffffff",
+    // @Field: ASraito: equivelent to true airspeed ratio
+    AP::logger().Write("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF,ASraito",
```

Comment:
Why not EAS2TAS?  same length, and it's what people kind of know this as.  It also conveys more information as "airspeed raito" (sic.) isn't really that descriptive.

I mean, the label is mispelled here, so it does need fixing anyway :-)

Reply:
Opps, I had already fixed the same typo in the commit message too.

Have have changed to `EAS2TAS` for some reason i thought it was longer. Your right that is better.

Thanks